lib/KBaseReport/KBaseReportImpl.pm

```diff
@@ -1,1048 +0,0 @@
-package KBaseReport::KBaseReportImpl;
```

Comment:
@rsutormin I didn't see this until after I submitted my own.  Can you take a look at mime?  I also removed some other dead stuff in the config.